src/main/java/de/numcodex/feasibility_gui_backend/query/v2/QueryHandlerRestController.java

```diff
@@ -148,9 +153,19 @@ public ResponseEntity<Object> getQuery(@PathVariable("id") Long queryId,
     return new ResponseEntity<>(query, HttpStatus.OK);
   }
 
+  @GetMapping("/{id}/result/with-sitenames")
```

Comment:
What about `/{id}/cleartext-result` and `/{id}/detailed-result` later? My variant uses one path segment for the result kind instead of like a combination of `result` and adjective of the result. Also works good together with the existing path segment of just `result` that is now more like a `summary result`. We could even implement a redirect for the old `result` path to `summary-result` and change the client in the long run.